librz/include/rz_search.h

```diff
@@ -146,9 +147,48 @@ typedef struct rz_search_interval_t RzSearchInterval;
 
 typedef struct rz_search_collection_t RzSearchCollection;
 
+typedef enum {
+	RZ_SEARCH_COLLECTION_HASH_ADLER32,
+	RZ_SEARCH_COLLECTION_HASH_ENTROPY_FRACT,
+	RZ_SEARCH_COLLECTION_HASH_ENTROPY,
+	RZ_SEARCH_COLLECTION_HASH_MD2,
+	RZ_SEARCH_COLLECTION_HASH_MD5,
+	RZ_SEARCH_COLLECTION_HASH_SHA1,
+	RZ_SEARCH_COLLECTION_HASH_SM3,
+	RZ_SEARCH_COLLECTION_HASH_XOR,
+	RZ_SEARCH_COLLECTION_HASH_CRC,
```

Comment:
There are many CRC algos. I would just push the name to RzHash instead of enums.